### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by jalvares-crown**

### CODE REVIEW

File: src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java [hunk 1]
```diff
@@ -65,11 +65,11 @@ public void setApiClient(ApiClient apiClient) {
      * @return Call to execute
      * @throws ApiException If fail to serialize the request body object
      */
-    public com.squareup.okhttp.Call organisationsOrganisationIdUsersGetCall(String organisationId, Integer pageSize, Integer currentPage, String searchString, Boolean includeSelf, final ProgressResponseBody.ProgressListener progressListener, final ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
+    public com.squareup.okhttp.Call organisationProfileorganisationIdUsersGetCall(String organisationId, Integer pageSize, Integer currentPage, String searchString, Boolean includeSelf, final ProgressResponseBody.ProgressListener progressListener, final ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
         Object localVarPostBody = null;
         
         // create path and map variables
-        String localVarPath = "/organisations/{organisationId}/users"
+        String localVarPath = "/organisation-profile{organisationId}/users"
```
Comment (ccs-wattsa) [inline src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java]: @jalvares-crown @tberey - is there a missing '/' between organisation-profile and {organisationId}?

File: src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java [hunk 2]
```diff
@@ -133,7 +133,7 @@ private com.squareup.okhttp.Call organisationsOrganisationIdUsersGetValidateBefo
 
     /**
      * Allows a user to retrieve users for a given organisation
-     * NOTE:- query params page-size, current-page  Sample request:                    GET /organisations/1/users?page-size&#x3D;10,current-page&#x3D;1
+     * NOTE:- query params page-size, current-page  Sample request:                    GET /organisation-profile1/users?page-size&#x3D;10,current-page&#x3D;1
```
Comment (ccs-wattsa) [inline src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java]: @jalvares-crown @tberey - missing / before the 1? Same issues on some lines below as well if so?

Comment (jalvares-crown) [reply src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java]: @ccs-wattsa  It looks like you have reviewed some old outdated files which don't exist in the latest version of the github repo. A good way to confirm this is to checkout the code and look for the line of code you have commented on. The Github interface can be misleading sometimes.

File: src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java (hunk 1 above)
Comment (jalvares-crown) [reply src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java]: @ccs-wattsa It looks like you have reviewed some old outdated files which don't exist in the latest version of the github repo. A good way to confirm this is to checkout the code and look for the line of code you have commented on. The Github interface can be misleading sometimes.

**Pull request closed without merging**